controllers/resource/reconciler.go

```diff
@@ -16,6 +17,11 @@ import (
 	anywhereTypes "github.com/aws/eks-anywhere/pkg/types"
 )
 
+const (
+	cpTemplatePrefix   = "-control-plane-template"
+	etcdTemplatePrefix = "-etcd-template"
```

Comment:
should we also use these constants in https://github.com/aws/eks-anywhere/blob/main/pkg/providers/common/common.go?